man/systemd.network.xml

```diff
@@ -2050,6 +2085,13 @@ Table=1234</programlisting></para>
           <ulink url="https://tools.ietf.org/html/rfc5227">RFC 5227</ulink>. Defaults to false.</para>
         </listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><varname>NetLabel=</varname></term>
+        <listitem>
+          <para>As in [Address] section.</para>
```

Comment:
this deserves a longer comment. people might misunderstand that this would enable respecting some field in the DHCP lease that configures a label. But this is not what it does, instead it just effects traffic to/from the IP address/network configured via DHCP on this iface get labelled with that is specified here. This should be configured explicitly

Reply:
Improved the comment